scss/bootstrap/overrides/_variables.scss

```diff
@@ -115,6 +115,11 @@ $crm-font-weight-h6:          500 !default;
 
 $icon-font-path: "../vendor/bootstrap/fonts/bootstrap/" !default;
 
+//== Layout
+//
+//## Common layout spacings, margins, and paddings
+
+$crm-gap-small: 5px !default;
```

Comment:
What is the need to add !default?

Reply:
`!default` means that if it's not defined, the value will be 5px, if it's defined, it will keep that value. This is used in case another theme wants to extend from this one. Most of the other variables have the `!default` suffix.

Reply:
I didn't think of "extending". Nice one.